packages/lib-classifier/src/plugins/drawingTools/experimental/components/LineControls/LineControls.js

```diff
@@ -0,0 +1,187 @@
+  let [activePosition, setActivePosition] = useState('tr')
+  const ARIA_LABEL = 'Line Controls'
+  const STROKE_COLOR = 'black'
+  const ARROW_STROKE_COLOR = 'white'
+  const FILL_COLOR = 'black'
+  const HOVER_COLOR = 'grey'
+  const STROKE_WIDTH = 1.5 / scale
+  const ARROW_STROKE_WIDTH = 0.5 / scale
+  const outerRadius = 40 / scale;
+  const innerRadius = 20 / scale;
+
+  const [x, y, w, h] = viewBox.split(' ').map(n => parseInt(n, 10))
+  const point = (activePosition == 'tr')
+    ? { x: x + (50 / scale), y: y + (50 / scale) }
+    : { x: x + w - (50 / scale), y: y + (50 / scale) }
+
+  const StyledPath = styled('path')`
+    fill: ${FILL_COLOR};
+    &:hover {
+      cursor: pointer;
+      fill: ${HOVER_COLOR};
+    }
+  `
+
+  const TL = `M ${point.x - outerRadius} ${point.y}
+    A ${outerRadius} ${outerRadius} 0 0 1 ${point.x} ${point.y - outerRadius}
+    L ${point.x} ${point.y - innerRadius}
+    A ${innerRadius} ${innerRadius} 0 0 0 ${point.x - innerRadius} ${point.y}
+    Z`;
+
+  const TR = `M ${point.x + outerRadius} ${point.y}
+    A ${outerRadius} ${outerRadius} 0 0 0 ${point.x} ${point.y - outerRadius}
+    L ${point.x} ${point.y - innerRadius}
+    A ${innerRadius} ${innerRadius} 0 0 1 ${point.x + innerRadius} ${point.y}
+    Z`;
+
+  const BL = `M ${point.x - outerRadius} ${point.y}
+    A ${outerRadius} ${outerRadius} 0 0 0 ${point.x} ${point.y + outerRadius}
+    L ${point.x} ${point.y + innerRadius}
+    A ${innerRadius} ${innerRadius} 0 0 1 ${point.x - innerRadius} ${point.y}
+    Z`;
+
+  const BR = `M ${point.x + outerRadius} ${point.y}
+    A ${outerRadius} ${outerRadius} 0 0 1 ${point.x} ${point.y + outerRadius}
+    L ${point.x} ${point.y + innerRadius}
+    A ${innerRadius} ${innerRadius} 0 0 0 ${point.x + innerRadius} ${point.y}
+    Z`;
+
+  const C = `M ${point.x - innerRadius} ${point.y}
+    A ${innerRadius} ${innerRadius} 0 0 0 ${point.x + innerRadius} ${point.y}
+    A ${innerRadius} ${innerRadius} 0 0 0 ${point.x - innerRadius} ${point.y}
+    Z`;
+
+  function move(e) {
+    if (activePosition == 'tl') {
+      setActivePosition('tr');
+    } else {
+      setActivePosition('tl');
+    }
+  }
+
+  function destroy() {
+    onDelete();
+  }
+
+  function onEnterOrSpace(ev, func) {
+    if (ev.keyCode === 13 || ev.keyCode === 32) func()
+  }
+
+  return (
+    <g
+      ref={ref}
+      role='group'
+      aria-label={'Line Controls'}
+      focusable='false'
+    >
+      <g
+        role='button'
+        aria-label='Undo'
+        focusable='true'
+        tabIndex='0'
+      >
+        <Tooltip label='Undo'>
+          <StyledPath
+            d={TL}
+            stroke={ARROW_STROKE_COLOR}
+            strokeWidth={ARROW_STROKE_WIDTH}
+            onPointerDown={mark.undo}
+            onKeyDown={(ev) => { onEnterOrSpace(ev, mark.undo) }}
+          ></StyledPath>
+        </Tooltip>
+      </g>
+      <g
+        role='button'
+        aria-label='Redo'
+        focusable='true'
+        tabIndex='1'
```

Comment:
```suggestion
        tabIndex='0'
```

Reply:
Setting the tabindex to 1 here means that this button is now after the end of the page, in the document tab order. Easily tested by tabbing through the page in a desktop browser.